docs/style.css

```
@@ -0,0 +1,662 @@
+    100% { top: 101%; }
+}
+
+/* Phosphor flicker - subtle brightness pulse */
+@keyframes phosphor-flicker {
+    0%, 100% { background: rgba(0, 0, 0, 0); }
+    50% { background: rgba(0, 0, 0, 0.03); }
+}
+
+/* VHS tracking - rare horizontal distortion band */
+@keyframes vhs-tracking {
+    0%, 92% { top: -10px; opacity: 0; }
```

Stylelint (17.2.0)

[error] Expected no more than 1 declaration (declaration-block-single-line-max-declarations)

(declaration-block-single-line-max-declarations)